cleanlab/datalab/internal/issue_finder.py

```diff
@@ -356,4 +342,7 @@ def get_available_issue_types(self, **kwargs):
                 )
                 issue_types_copy.pop("outlier")
 
+        if self.task == "regression":
+            issue_types_copy = {**issue_types_copy, **{"label": {}}}
```

Comment:
Maybe add a comment above here why `"label"` is being "reset".

Can you confirm that this line won't run if `"label"` is not in the issue_types dictionary?

Reply:
Detailed explanation is here: https://github.com/cleanlab/cleanlab/pull/796#discussion_r1316764067
I've added addition comment in code